xbmc/epg/Epg.cpp

```diff
@@ -52,9 +50,7 @@ CEpg::CEpg(int iEpgID, const CStdString &strName /* = "" */, const CStdString &s
   m_pvrChannel = empty;
 }
 
-CEpg::CEpg(CPVRChannelPtr channel, bool bLoadedFromDb /* = false */) :
-    m_bChanged(!bLoadedFromDb),
-    m_bTagsChanged(false),
+CEpg::CEpg(CPVRChannelPtr channel) :
```

Comment:
Using a const CPVRChannelPtr & would be better here. No need for passing data "by value", to construct another CPVRChannelPtr instance.